walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -1,39 +1,40 @@
 // tag::master-1[]
-:walkthrough: Integrating event-driven and API-driven applications (AMQ)
+:walkthrough: Integrating message-oriented middleware with an API using AMQ
+:messaging-service: Red Hat AMQ
 :fuse-version: 7.2
+:AMQ-ProductLongName: Red Hat AMQ
+:AMQ-BrokerVersion: 7.2
+:Fuse-prodnamefull: Red Hat Fuse
+:node-url: http://frontend-node-app.apps.city.openshiftworkshop.com/
+:spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
+:fuse-url: https://eval.apps.city.openshiftworkshop.com/
+
 
 
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id='integrating-eventdriven-and-apidriven-applications']
-// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules-{context}'].
 
-// Book Title
 = {walkthrough}
 
+// word count range that fits best is 15-22, with 20 really being the sweet spot. Character count for that space would be 100-125
+This walkthrough demonstrates a pattern that integrates a message driven architecture with an API driven architecture. 
```

Comment:
Maybe add ``` REST API ```?

Reply:
"...with an API driven architecture" => "...with an API driven architecture using Fuse Online as the integration platform"

Reply:
@maleck13  how about this:
This walkthrough demonstrates a pattern that integrates a message driven architecture with a REST API driven architecture using Fuse Online.

Reply:
👍also good, though I'd change "REST API driven architecture" to "RESTful architecture using..." or "RESTful web services using..."

The more I think actually the more I tend towards "This walkthrough demonstrates a fault tolerant integration pattern between RESTful web services using a message-driven architecture and Fuse Online."

Am I going to far out. It seems important to me to include "integration" and "fault tolerance".

Reply:
FYI: JOR has asked me to not mention HTTP (the protocol) and to use RESTful API as the term

Reply:
also JOR asked not use product names upfront, that's the implementation